gnd/build.gradle

```diff
@@ -68,6 +68,9 @@ android {
         }
         debug {
             resValue("string", "GOOGLE_MAPS_API_KEY", googleMapsApiKey)
+            FirebasePerformance {
```

Comment:
I didn't realize you could disable it this way, thanks!

One question - for debugging the app, wouldn't we also want this enabled? If so, should we instead have a different [default] variant, e.g. `dev`, for use during development?

Reply:
I guess not. It would be most useful once the app is released.

Reply:
Ok, let's do this for now then. Thanks!